tests/acceptance/test_dbus.py

```diff
@@ -109,6 +109,13 @@ def test_dbus_get_jwt_token(self, bitbake_variables, connection, setup_mock_serv
             assert f'string "{self.JWT_TOKEN}' in output
             if version_is_minimum(bitbake_variables, "mender-client", "3.3.1"):
                 assert 'string "http://127.0.0.1:' in output
+                # Check that this really is the only port we're listening on.
+                lines = connection.run("netstat -t -u -l -p -n").stdout.split("\n")
```

Comment:
I once went down this road, and then found [this one](https://github.com/mendersoftware/integration/blob/master/tests/tests/test_keepalive.py#L135) an alternative. have you considered doing something like:

```bash
cat /proc/`pidof mender`/net/tcp | grep -E '[^:]+: [^ ]+ [^ ]+ 0A '
```

Reply:
Is it better though? The proc file needs to be decoded before it matches the string.

Reply:
> Is it better though? The proc file needs to be decoded before it matches the string.

I said "an alternative" on purpose :) I do not know about better, advantages:

* explicitly checks the ports in the listening state for given process
* parsing a file maybe seen as more elegant solution than running netstat and parsing output looking for /mender

do you think you can make it work with /proc parsing?

Reply:
Well, my command also checks for a given process. As for parsing a file, well, maybe, if we assume that the output from `netstat` is unpredictable depending on the exact version for example. But I don't think this is the case, and using the file is quite cumbersome since it is based on hex. When using hex values, neither the port number nor the address is greppable in the source code, and it is hard to read as well. In addition, my command also checks UDP.

I vote for simplicity in this case. If it turns out that `netstat` output changes incompatibly during the next build host upgrade, then you have my official permission to say: I told you so! 🙂

Reply:
@merlin-northern: With that in mind, do you think it is worth an approval? :pray:

Reply:
one thing about me: no matter what I never say: "I told you so" :) ok, let do it the netstat way.